docker-compose/README.md

```diff
@@ -115,6 +115,7 @@ The result of the invokation should be printed on the terminal:
 - OpenWhisk Controller - `~/tmp/openwhisk/controller/logs/`
 - OpenWhisk Invoker - `~/tmp/openwhisk/invoker/logs/`
 - `docker-compose` logs - `~/tmp/openwhisk/docker-compose.log`
+- Action output such as stdout or console.log(): `wsk -i activation logs <activationId>`
```

Comment:
WDYT if we also cross-reference this tutorial : https://github.com/IBM-Bluemix/openwhisk-workshops/tree/master/bootcamp#start-your-engines , for more info, in addition to this quick command that you found useful.

Reply:
+1 seems like a helpful resource for me as well :-)

Reply:
done in section above